content/en/docs/concepts/services-networking/service.md

```diff
@@ -183,6 +183,13 @@ Accessing a Service without a selector works the same as if it had a selector.
 In the example above, traffic is routed to the single endpoint defined in
 the YAML: `192.0.2.42:9376` (TCP).
 
+{{< note >}}
+The Kubernetes API server does not allow proxying to endpoints that are not mapped to pods. Actions such as
+`kubectl proxy <service-name>` where the service is without a selector will fail due to this 
+constraint. This is due to security constraint where api-server can be used to proxy to external
+endpoints where the caller may not have access to but api-server does. 
```

Comment:
you only replaced one api-server by `Kubernetes API server` , this lines still have `api-server does`